assets/scss/modules/_images.scss

```diff
@@ -23,54 +22,63 @@
   background-repeat: no-repeat;
 }
 
-.icon-star {
-  @include icon-table;
-  background-image: url('../../img/star.svg');
-}
+.dp-calification {
+  display: flex;
+  align-items: center;
 
-.icon-star-filled {
-  @include icon-table;
-  background-image: url('../../img/star-filled.svg');
+  .icon-star {
+    @include icon-table;
+    color: $dp-color-yellow;
+  }
+
+  .icon-star-filled {
+    @include icon-table;
+    color: $dp-color-silver;
+  }
 }
 
+///
+/// Icons for Subscriber History
+///
+
 // User active
-.icon-user--active:before {
+.user-active:before {
   @include icon-user-table;
   background-image: url('../../img/user-active.svg');
 }
 
 // User active not associated to lists
-.icon-user--natl:before {
+.user-active-with-no-list:before {
```

Comment:
Are we not using BEM here for class naming? I think here perhaps `user--active-with-no-list` would be best to represent element with modifier. What do you think?